zh/deploy-on-aws-eks.md

```diff
@@ -374,9 +393,21 @@ parameters:
   fsType: ext4
```

Comment:
Could you please add other parameters, e.g. iops for the io1 sc?

Reply:
https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs

Reply:
added in following commit